File: core/src/main/java/com/rigoloapp/alenandria/Alejandria.java
package com.rigoloapp.alenandria;

import com.badlogic.gdx.Game;

/** {@link com.badlogic.gdx.ApplicationListener} implementation shared by all platforms. */
public class Alejandria extends Game {
	@Override
	public void create() {
		setScreen(new FirstScreen());
	}
}

File: core/src/main/kotlin/com/rigoloapp/alejandria/Alejandria.kt
package com.rigoloapp.alejandria

import com.rigoloapp.alejandria.screen.AlejandriaScreen
import com.rigoloapp.alejandria.screen.TestingLevelScreen
import ktx.app.KtxGame

/** [com.badlogic.gdx.ApplicationListener] implementation shared by all platforms.  */
class Alejandria : KtxGame<AlejandriaScreen>() {
    override fun create() {
        addScreen(TestingLevelScreen(this))
        setScreen<TestingLevelScreen>()
    }
}